images/tensorflow-notebook/cuda/nvidia-lib-dirs.sh

```diff
@@ -2,8 +2,8 @@
 # Copyright (c) Jupyter Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-# This adds the NVIDIA libraries to the LD_LIBRARY_PATH. Workaround for
-# https://github.com/tensorflow/tensorflow/issues/63362
+# This adds NVIDIA Python package libraries to the LD_LIBRARY_PATH. Workaround
+# for https://github.com/tensorflow/tensorflow/issues/63362
 NVIDIA_DIR=$(dirname "$(python -c 'import nvidia;print(nvidia.__file__)')")
 LD_LIBRARY_PATH=$(echo "${NVIDIA_DIR}"/*/lib/ | sed -r 's/\s+/:/g')${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
 export LD_LIBRARY_PATH
```

Comment:
Note that this is something else than whats done to make `nvidia-smi` work, so I've made the comment above a bit more explicit to say that this was related to the nvidia python package.